component/toast/toast.css

```diff
@@ -0,0 +1,43 @@
+.component_box.toast_box {
+    height: 10rem;
+}
+
+.container {
+    width: 25rem;
+    position: relative;
+    bottom: 1.5rem;
+}
+
+#SnackBar {
+    display: flex;
+    justify-content: space-between;
+    background-color: var(--danger-alert-bg);
+    border-radius: 0.4rem;
+    font-size: 1.5rem;
+    padding: 1.5rem;
+    color: var(--danger-alert-color);
+}
+
+i#btn_close {
+    color: var(--black-color);
+    cursor: pointer;
+}
+
+.btn {
+    padding: 0.5rem;
+    margin-left: 1rem;
+    font-size: 1.1rem;
+    border: none;
+    cursor: pointer;
+    color: var(--danger-alert-color);
+    background-color: var(--black-color);
+}
+
+.snackBar {
+    background-color: var(--black-color);
+    border-radius: 0.4rem;
```

Comment:
Since you will be needing the border radius properties at many places, it's better to create 2-3 variables for border-radius and use only those values, so that every component maintains consistency. Otherwise every component might get different different values for border-radius

Reply:
thank you for reviewing. yes, I will shift this to a utility class.